### PR TITLE
[plg_system_updatenotification]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_system_updatenotification
### Testing Instructions

code review

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on ba24f61de7ad53f4f152d82ac3044fe4c1f50450<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10506">issues.joomla.org/joomla-cms/10506</a>.</sub>

File: plugins/system/updatenotification/updatenotification.php [hunk 1]
```diff
@@ -217,7 +217,7 @@ public function onAfterRender()
 		$mailFrom = $jConfig->get('mailfrom');
 		$fromName = $jConfig->get('fromname');
 
-		$substitutions = array(
+		$substitutions     = array(
```
Comment (andrepereiradasilva) [inline plugins/system/updatenotification/updatenotification.php]: why the space?

Comment (wojsmol) [inline plugins/system/updatenotification/updatenotification.php]: align with lines below

Comment (andrepereiradasilva) [inline plugins/system/updatenotification/updatenotification.php]: what lines below?

Comment (wojsmol) [inline plugins/system/updatenotification/updatenotification.php]: https://github.com/wojsmol/joomla-cms/blob/plg_system_updatenotification-cs/plugins/system/updatenotification/updatenotification.php#L220-L226

Comment (andrepereiradasilva) [inline plugins/system/updatenotification/updatenotification.php]: i would understand aligning with lines above, not with below.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 5cc7c08a76d367fa582e7bb90fd991e016652296<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10506">issues.joomla.org/joomla-cms/10506</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 5cc7c08a76d367fa582e7bb90fd991e016652296<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10506">issues.joomla.org/joomla-cms/10506</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10506">issues.joomla.org/joomla-cms/10506</a>.</sub>

**Pull request merged**